File: Listas/Lista2/Exercícios_python/Sala/4.py
a = float(input("Entre com o valor do lado A: "))
b = float(input("Entre com o valor do lado B: "))
c = float(input("Entre com o valor do lado C: "))

if a<(b+c) and b<(a+c) and c<(a+b) and a>0 and b>0 and c>0:
    if a==b and b==c:
        print("Os valores fornecidos remetem a um triângulo equilátero.")
    elif a==b or b==c or c==a:
        print("Os valores fornecidos remetem a um triângulo isósceles.")
    else: 
        print("Os valores fornecidos remetem a um triângulo escaleno.")
else:
    print("Esses valores não formam um triângulo.")

File: Listas/Lista2/Exercícios_C/Sala/4.c
#include <stdio.h>
int main ()

{
    float a, b, c;
    printf("Entre com o valor de A: ");
    scanf("%f", &a);
    printf("Entre com o valor de B: ");
    scanf("%f", &b);
    printf("Entre com o valor de C: ");
    scanf("%f", &c);

    if (a<(b+c) && b<(a+c) && c<(a+b) && a>0 && b>0 && c>0) {
        if (a==b && b==c) {
            printf("Os valores fornecidos remetem a um triângulo equilátero.");
        }
        else if (a==b || b==c || c==a) {
            printf("Os valores fornecidos remetem a um triângulo isósceles.");
        }
        else {
            printf("Os valores fornecidos remetem a um triângulo escaleno.");
        }
    }
    else {
        printf("Esses valores não formam um triângulo.");
    }
}
